tests/ClientTest.php

```diff
@@ -1846,15 +1836,7 @@ public function testEncodeTooDepth()
             $data_broken = [$data_broken];
         }
         $value = $client->encode($data_broken);
-        if (version_compare(PHP_VERSION, '5.5.0', '>=')) {
-            $this->assertFalse($value, 'Broken data encoded successfully with native method');
-        } else {
-            if ($value !== false) {
-                $this->markTestSkipped();
-            } else {
-                $this->assertEquals('eJyLjh4Fo2AUjFgQOwpGwSgYuQAA3Q7g1w==', $value, 'Native json_encode error');
-            }
-        }
+        $this->assertFalse($value, 'Broken data encoded successfully with native method');
```

Comment:
Didn't this removed too much code? I'm not sure the `else` part should be entirely removed